tests/sparseml/pytorch/sparsification/pruning/test_modifier_pruning_movement.py

```diff
@@ -0,0 +1,270 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
It seems like the structured pruning tests have been placed in the movement pruning test file and vice versa